src/components/Details/Details.tsx

```diff
@@ -1,7 +1,67 @@
-import React from 'react';
+import React from "react";
+import "../Details/Details.css"
 
-interface DetailsProps {}
+interface MarketProps {
+  markets: {
+    id: number;
+    distanceFromZip: number;
+    marketName: string
+  }[],
+  marketDetails: {
+    street: string,
+    city: string,
+    state: string,
+    zip: string,
+    schedule: {
+      dayOfWeek: string,
+      time: string,
+      season: string
+    }[],
+    products: string[],
+    mapsLink: string
+  }[],
+  id: string;
+}
+
+const Details: React.FC<MarketProps> = ({ id, markets, marketDetails }) => {
+
+  const marketMatch = markets.filter(market => market.id === Number(id));
+  const detailsMatch = marketDetails.filter(market => { 
+    let match = market.mapsLink.split('%22')[1].includes(marketMatch[0].marketName.split(' ')[0]);
+    console.log(marketMatch[0].marketName)
+    return match
+  });
+  const nameMatch = marketMatch[0].marketName;
+  const detail = detailsMatch[0];
+  const openSeason = `Season: ${detail.schedule[0].season}`;
+  const daysAndTimes = `Open: ${detail.schedule[0].dayOfWeek} ${detail.schedule[0].time}`;
+  const productList = detail.products.map(prod => {
+    return (
+    <div className='list'>
+      <ul>
+        <li> {prod}, </li>
+      </ul>
+    </div>
+    );
+  });
+
+  return (
+    <section className='market-details'>
+      <h1> {nameMatch} </h1>
+        <p> Located at: {detail.street}, {detail.city} {detail.state}, {detail.zip} </p>
+      <div className='schedule'>
+        <h4> Season and Schedule: </h4>
```

Comment:
I think best practice is no whitespace between html elements. I.e.: `<h4>Season and Schedule:</h4>`
[Turing style guide](https://github.com/turingschool-examples/html)

Reply:
wow! personally I find it's incredibly hard to read so I've done it for my own readability but haven't specifically been given feedback regarding that in the past - so I appreciate you bringing this to my attention. All fixed.